src/config.js

```diff
@@ -9,19 +11,24 @@ var config = require('rc')("login-dc", {
     }
 })
 
-if (!config.email_address || !config.email_password) {
-    console.error("Missing configuration: Email address or password is missing.")
-    process.exit(1)
+const configFile = config.config || `./.${appName}rc`
```

Comment:
what are you doing with this line?, I don't fully understand its existance

Reply:
at least I think the config file name would be `~/.${appName}.rc`, not `./.${appName}rc`? so in ~, and with a . between appName and rc?

Reply:
At least in production, the config file is in the home directory.